### ISSUE TITLE
Plurality issue in the URI listing in the replay UI

### ISSUE THREAD

**Issue by machawk1** [oduwsdl/ipwb#534]
![screen shot 2018-08-27 at 11 57 17 am](https://user-images.githubusercontent.com/2514780/44670365-7a114f00-a9f0-11e8-84b8-d67c8b82af7c.png)

It should be "1 memento of 1 resources with 0 HTML pages listed"

**Comment by machawk1** [comment]
With the new templating system, we may want to move the logic in `setPlurality()` (JS) to attributes of the `showLandingPage()` in replay.py. This should make these values display more consistently regardless of the client's JS rendering.
